package co.udea.certificacion.moduloseguimientodeordenes.exceptions;

public class Exception {
    private Exception(){}
}